Week1/homework/Masoud/js-exercises/EX10-compareArrays.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+let animal = ["birds", "monkeys", "cats", "dogs"];
+let oddNum = [1, 3, 5, 7, 9, 11, 13];
+console.log("The length of the array is " + oddNum.length);
+console.log("The length of the array is " + animal.length);
+
+if(animal.length == oddNum.length){console.log("They are the same!");}
```

Comment:
Use proper formatting and spacing and be consistent. Pick one style and use that. you can configure your VSCode or your editor to help you indent automatically.

Some resource: https://courses.cs.washington.edu/courses/cse154/18au/resources/styleguide/js/spacing-indentation-js.html